content/blog/spring-k8s-cassandra.md

```diff
@@ -0,0 +1,451 @@
+---
+title: "Save Your Stack: Build Cloud Native Apps with Spring, Kubernetes and Cassandra"
+featured: true
+weight: 3
+description: >
+    Build modern, cloud native apps today using the latest innovations from Spring, Cassandra and Kubernetes
+date: 2020-07-08
+topics:
+- Spring
+- Containers
+- Kubernetes
+tags:
+- Containers
+- API
+- Spring
+- Spring Cloud Gateway
+- Spring Boot
+- Spring Cloud
+- Kubernetes
+patterns:
+- API
+# Author(s)
+team: 
+- Chris Splinter
+---
+
+> This blog was written by Chris Splinter who works for DataStax, focusing on open source solutions in the Apache Cassandra ecosystem. As a former automation engineer, Chris is focused on developers and making it easier to build applications through APIs, frameworks, and tools. You can find him on [LinkedIn](https://www.linkedin.com/in/csplinter/).
```

Comment:
I think this works! I wonder about putting it on the bottom instead even though it is a long post.

Reply:
That works too, I was debating if it'd be better at the top or bottom so I'm not too picky